Updated contributing.md (#7)
mlsanigeria/AI-Hacktober-MLSA: CONTRIBUTING.md

4. Work on your image classification tasks in your notebook.

> 5. The minimum contribution a user can make is data processing.

Reviewer: ```suggestion
5. The minimum contribution you can make is on preprocessing of the data(images).
```

6. For contributions related to modeling, the model's performance must achieve a minimun of 60% accuracy to be considered valid.

> 7. The team will conduct test on model using a specific dataset to verify its performance score.

Reviewer: ```suggestion
7. The team will perform an evaluation of the model using a separate test dataset to verify its performance score.
```

> 7. The team will conduct test on model using a specific dataset to verify its performance score.

Reviewer: lkml
